cookbook/integrations/aws/in_container.mk

```diff
@@ -0,0 +1,24 @@
+SERIALIZED_PB_OUTPUT_DIR := /tmp/output
```

Comment:
Can we mention somewhere in the README that this file should be at one level above the folder that contains the DockerFile. ie. the docker context is currently defined in common/Makefile in docker_build.
May be we can that a variable to be explicit and have the folder structures follow that convention.